client/components/theme/index.jsx

```diff
@@ -137,6 +149,7 @@ const Theme = React.createClass( {
 				<div className="theme__content">
 					{ this.renderHover() }
 					<a href={ this.props.screenshotClickUrl }>
+						{ this.renderInstalling() }
```

Comment:
I'd be tempted to go with `{ installing && this.renderInstalling() }` here, meaning we could lose an indent level inside `renderInstalling()`, but there is nothing wrong with it as is.